### PR TITLE
feat(build): provide installation via Scoop (Windows)

### PR DESCRIPTION
**Pull request by sanderblue**
Resolves: #167 

This PR adds Scoop to the GoReleaser, which will generate and publish a Scoop App Manifest into our repository. Windows users should be able to install the CLI via `scoop`.

The README has also been updated with installation instructions for Scoop.

Installing with Scoop:
```bash
scoop bucket add newrelic-cli https://github.com/newrelic/newrelic-cli.git
scoop install newrelic-cli
```

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -99,3 +99,17 @@ snapcrafts:
...
         # https://snapcraft.io/docs/reference/interfaces.
         plugs: ["home", "network"]
+
+scoop:
+  bucket:
+    owner: newrelic
+    name: newrelic-cli
+  commit_author:
+    name: nr-developer-toolkit
+    email: developer-toolkit-team@newrelic.com
```
Comment (jthurman42) [inline .goreleaser.yml:109]: We don't want folks emailing the team directly, use: `opensource@newrelic.com`

Comment (sanderblue) [reply .goreleaser.yml]: Used a secret instead. Should be obfuscated in CI now 😎

File: README.md [hunk 2]
```diff
@@ -68,6 +68,12 @@ Silent installation of the latest version of the CLI can be achieved via the fol
 msiexec.exe /qn /i "$env:TEMP\NewRelicCLIInstaller.msi" | Out-Null; `
 ```
 
+#### Scoop
```
Comment (ctrombley) [inline README.md:71]: Can we move this to the top of the Windows section?

Comment (sanderblue) [reply README.md:71]: Sure thing

Comment (sanderblue) [reply README.md]: Done

**Pull request merged**